Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -1342,6 +1366,8 @@ theorem trans_transPartialHomeomorph (e : α ≃ₜ β) (e' : β ≃ₜ γ) (f''
 
 end Homeomorph
 
+section OpenEmbedding
+
```

Comment:
wrapping a namespace in a section with no extra bits in between, *especially* when they have the same name is just confusing.
```suggestion
```

Reply:
My reasoning was that sections can be collapsed in VS Code, making it easier to see at a glance what's in the file. However, namespaces have the same effect, so I'm convinced. Changed (here and elsewhere).